docs/self-hosted/install/run-as-docker.md

```diff
@@ -0,0 +1,116 @@
+---
+url: /docs/self-hosted/latest/install/run-as-docker/
+---
+
+# Running Gitpod as a Docker container
+
+For smaller setups or to test Gitpod Self-Hosted on your local machine you can run a complete Gitpod installation as a Docker container. The Gitpod Docker image [`gitpod-k3s`](https://console.cloud.google.com/gcr/images/gitpod-core-dev/EU/build/gitpod-k3s) is based on the [`k3s` Docker image](https://hub.docker.com/r/rancher/k3s).
+
+## Running Gitpod using `docker run`
+
+The easiest way to get Gitpod up with `docker run` is as follows:
+
+```
+$ docker run \
+    --privileged \
+    --name gitpod \
+    --publish 443:443 --publish 80:80 \
+    --env DOMAIN=10-0-0-5.ip.mygitpod.com \
+    eu.gcr.io/gitpod-core-dev/build/gitpod-k3s
+```
+
+Make sure to replace `10-0-0-5` with the IP address of your machine (replace dots with dashes). [See below](#customize-your-gitpod-installation) how to customize your installation (e.g. with an own domain).
+
+## Running Gitpod using `docker-compose`
+
+To run Gitpod you can also use our sample [`docker-compose.yaml`](https://github.com/gitpod-io/gitpod/tree/clu/docker-k3s-docs/install/docker/examples/gitpod/docker-compose.yaml) file.
```

Comment:
Better to use not absolute urls, but relative, 
Instead of https://github.com/gitpod-io/gitpod/tree/clu/docker-k3s-docs/install/docker/examples/gitpod/docker-compose.yaml which go to your specific branch, which will be removed after PR will be merged
you can use ../../../install/docker/examples/gitpod/docker-compose.yaml